util/runner.py

```diff
@@ -0,0 +1,167 @@
+import logging
+from mpi4py import MPI
+
+from util.logger import SCLogger
+
+class Runner:
```

Comment:
General comment/question on this: It looks to me like this is much more complicated than using python's multiprocessing. What are the advantages of doing it this way?

Reply:
We will probably not actually use this functionality, but this works across machines, as MPI can work across machines.  (I imported this from ... curveball? another pipeline.  I used it in `hacks/rknop/process_decam_exposure_mpi.py` when I was exploring the possibility that some dysfunctionalities were arising from using python multiprocessing rather than MPI (which is what I'd always used in pipelines in the past).  The conclusion was that that was not the problem.  I left this in so the hacks mpi thing would still work; eventually, we may throw this out.

(Also, the only reason using MPI is more complicated is that multiprocessing provides the Pool class.  I originally reimplemented this same functionality using multiprocessing, and it was approximately as complicated, until I smacked myself in the face and remembered the Pool class.)